jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/cpu/OverallCPULoadHandler.java

```diff
@@ -27,52 +26,36 @@
   private static final String JVM_SYSTEM = "jvmSystem";
   private static final String MACHINE_TOTAL = "machineTotal";
 
-  private static final String METRIC_NAME = "runtime.jvm.cpu.utilization";
-  private static final String DESCRIPTION = "CPU Utilization";
```

Comment:
Similar comment as to METRIC_NAME - why are we inlining the description constant?